package.json

```diff
@@ -23,6 +23,7 @@
     "@types/react": "^18.0.17",
     "@types/react-dom": "^18.0.6",
     "bootstrap": "^5.2.0",
+    "bootstrap-icons": "^1.9.1",
```

Comment:
check if it's required or can be deleted

Reply:
done